interface/forms/newpatient/newpatient.js

```diff
@@ -191,6 +191,14 @@ window.NewPatientForm = function(window) {
             }
         }
 
+        //Show confirmation alert when the form is submitted
+        const form = document.getElementById('new-encounter-form');
+        if (form) {
+            form.addEventListener('submit', function() {
+                alert(xl('Form submitted successfully!'));
```

Comment:
alerts force a click when staff is juggling many tasks. If a feedback is must, consider bootstrap's of alert classes. Unlike js alert box, you can even include other html elements to make it part of a workflow. Add an timer and the alert can optionally close itself after n seconds.